library_schemas/score/prerelease/HED_score_1.0.0.mediawiki

```diff
@@ -984,17 +984,14 @@ This project was supported by the by the National Institute of Mental Health of
             *** Clinical-EEG-temporal-relationship
                 **** Clinical-start-followed-EEG <nowiki>[Clinical start, followed by EEG start by X seconds.]</nowiki>
                     ***** <nowiki># {takesValue, valueClass=numericClass, unitClass=timeUnits}</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[free text]</nowiki>
```

Comment:
Clinical-start-followed-EEG can take a numeric value to note that there is a clinical start, followed by EEG start by X seconds AND free text can be added.

How should this be enabled in the schema?

Reply:
HED doesn't allow this.  It is technically possible to have allow multiple value classes in the same node, so you could have:

`***** <nowiki># {takesValue, valueClass=numericClass, valueClass=textClass, unitClass=timeUnits}</nowiki>`

but you couldn't use both types of values at the same time. You would need to use two tags:

`(Clinical-start-followed-EEG/1s, Clinical-start-followed-EEG/These are my notes)`

I think this would be a mistake.  It would be hard for the validators to correctly detect errors.

**I would strongly recommend instead adding:**

```code
*** Clinical-EEG-temporal-relationship-notes <nowiki>[Clinical notes to annotate the clinical-EEG temporal relationship.]</nowiki>
***** <nowiki># {takesValue, valueClass=textClass}</nowiki>
```

The notes tag could be used with any of the other tags in that subtree and solve the problem.

Reply:
The notes tag in the subtree makes sense to me.
Would you be able to change it in this PR?

Reply:
Done.